backend/src/StdLibExecution/Libs/Bool.fs

```diff
@@ -55,40 +40,6 @@ let fns : List<BuiltInFn> =
         | _ -> incorrectArgs ())
       sqlSpec = SqlBinOp "OR"
```

Comment:
When the stdlib functions are moved to packages, what will happen to the sqlSpec?"